Build/build-pack-nano-nugets.psm1

```diff
@@ -2,31 +2,13 @@
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\Frequency\UnitsNet.NanoFramework.Frequency.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\Illuminance\UnitsNet.NanoFramework.Illuminance.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\Length\UnitsNet.NanoFramework.Length.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\Pressure\UnitsNet.NanoFramework.Pressure.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\Ratio\UnitsNet.NanoFramework.Ratio.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\RelativeHumidity\UnitsNet.NanoFramework.RelativeHumidity.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\Temperature\UnitsNet.NanoFramework.Temperature.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\VolumeConcentration\UnitsNet.NanoFramework.VolumeConcentration.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\Power\UnitsNet.NanoFramework.Power.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\Density\UnitsNet.NanoFramework.Density.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\Mass\UnitsNet.NanoFramework.Mass.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\Speed\UnitsNet.NanoFramework.Speed.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\MassConcentration\UnitsNet.NanoFramework.MassConcentration.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\Angle\UnitsNet.NanoFramework.Angle.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\MagneticField\UnitsNet.NanoFramework.MagneticField.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\Acceleration\UnitsNet.NanoFramework.Acceleration.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir"
-  & $nuget pack "$root\UnitsNet.NanoFramework\GeneratedCode\Level\UnitsNet.NanoFramework.Level.nuspec" -Verbosity detailed -OutputDirectory "$nugetOutDir" 
-
+Foreach ($nuspecFile in $nugetsToProcess)
```

Comment:
When the nanoFramework builds for Units.Net  started being published the decision was to add packages to release as they are needed. Mostly because there is not enough manpower to test everything. 
All the nanoFramework NuGet seem to be performing just fine. At
least there are no outstanding complains. The build & publish process has been fine-tuned and is now quite mature. 
This is to say that I don't have anything against publishing them all, if that's Anders decision.

Final note: the build automation is only capable to handle the auto-reference of the Core lib and Math packages. If any of the packages requires reference to any other package, it has to be left out or the processed adjusted accordingly.

Reply:
Yes but I am missing packages.
I think it's better to release it wrong dependencies, and have it bug reported, than to hoarde all packages, and slowly test them. It's not just core nanoframework team who uses these, I am migrating my own drivers for it, and I need different packages.

Reply:
Looks good to me, let's try it. I have no problems publishing them all. We'll address any issues that arise and are reported for the new packages.